configs/navigate.go

```diff
@@ -1,37 +0,0 @@
-/*
- * This file is part of arduino-cli.
- *
- * Copyright 2018 ARDUINO SA (http://www.arduino.cc/)
- *
- * This software is released under the GNU General Public License version 3,
- * which covers the main part of arduino-cli.
- * The terms of this license can be found at:
- * https://www.gnu.org/licenses/gpl-3.0.en.html
- *
- * You can be released from the requirements of the above licenses by purchasing
- * a commercial license. Buying such a license is mandatory if you want to modify or
- * otherwise use the software for commercial activities involving the Arduino
- * software without disclosing the source code of your own applications. To purchase
- * a commercial license, send an email to license@arduino.cc.
- */
-
-package configs
-
-import (
-	paths "github.com/arduino/go-paths-helper"
-	"github.com/sirupsen/logrus"
-)
-
-// Navigate FIXMEDOC
-func (c *Configuration) Navigate(pwd *paths.Path) {
```

Comment:
Does viper provides a `Navigate` feature already?

Reply:
You can call `viper.AddConfigPath()` several times passing the path where Viper should look for a config file